packages/core/networks/82.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "contractName": "Finder",
+    "address": "0x74f0D6b0Ea5B142052ca271f0087af20333304Bd"
```

Comment:
Verification? https://scan.meter.io/address/0x74f0D6b0Ea5B142052ca271f0087af20333304Bd?tab=2&p=1

Reply:
I have created a task that I will complete after the audit